### ISSUE TITLE
Excuse me, demo can not run, how to solve?

### ISSUE THREAD

**Issue by StarRain-L** [lionheartX/tensorflow-on-ios-tutorial#1]
![ba3f6389-6cbf-407a-927a-1a050b5a7ae1](https://cloud.githubusercontent.com/assets/8908244/26721369/2a750c4c-47be-11e7-81b6-a7d5aa118fb0.png)

This configuration path, the team how to share???

**Comment by lionheartX** [comment]
yeah you need to change the path to point to your documents ( so change this part /Users/zhengxiong/Development/)

**Comment by StarRain-L** [comment]
谢谢,,,我还有个问题,这个.a文件这么大,怎么减少呢?感谢回复
![12cf3dff-1008-4fbf-8c82-b3adb8a00a49](https://cloud.githubusercontent.com/assets/8908244/26774889/063a1428-4a05-11e7-8e68-2f9c48ce4b5c.png)
